frontend/src/app/keyresult-detail/keyresult-detail.component.html

```diff
@@ -0,0 +1,65 @@
+<div class="d-flex flex-column" cdkTrapFocus="true" *ngIf="keyResult">
+  <div class="d-flex justify-content-between">
+    <h1 class="mb-2 title">{{ keyResult.title }}</h1>
+  </div>
+  <div class="keyresult-header d-flex justify-content-between mb-3">
+    <div>
+      {{ "KEY_RESULT_Type." + keyResult.keyResultType | translate }}
+    </div>
+    <div class="d-flex align-items-center">
+      <img src="assets/icons/person-icon.svg" width="10" height="10" class="me-1" alt="person icon" />
+      <p>Owner {{ keyResult.owner.firstname }} {{ keyResult.owner.lastname }}</p>
```

Comment:
change layout if metric for baseline and stretch goal, rename Stretch-Goal to Stretch Goal, Remove Owner prefix, add confidence label, change fontsize on buttons